interfaces/IAnalytics.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+
+#include "Module.h"
+
+// @stubgen:include <com/IIteratorType.h>
+
+namespace WPEFramework {
+namespace Exchange {
+    // @json 1.0.0
+    struct EXTERNAL IAnalytics : virtual public Core::IUnknown {
+        enum { ID = ID_ANALYTICS };
+
+        virtual ~IAnalytics() override = default;
+
+        using IStringIterator = RPC::IIteratorType<string, RPC::ID_STRINGITERATOR>;
+
+        
+        // @alt sendEvent
+        // @brief Send an event to the analytics server
+        // @param eventName: Name of the event
+        // @param eventVersion: Version of the event
+        // @param eventSource: Source of the event
+        // @param eventSourceVersion: Version of the event source
+        // @param cetList: List of CETs
+        // @param epochTimestamp: Epoch timestamp of the event
+        // @param uptimeTimestamp: Uptime timestamp of the event
+        // @param eventPayload: Payload of the event
+
+        virtual Core::hresult SendEvent(const string& eventName /* @in @text eventName*/,
```

Comment:
You can define @text:keep on interface level, than you do not need to use the @text on all paramaters. Might be easier to maintain..

See https://rdkcentral.github.io/Thunder/plugin/interfaces/tags/  

Snippit taken from there: "When used for a whole interface it must be used in the form '@text:keep'. It will in this case make the JSON generator use the name of the above items in JSON code exactly as specified in the interface, including the case of the text. Please note that of course the interface designer is responsible for making the interface compliant and consistent with the interface guideliness in use (even more perhaps then with the other uses of @text as now the whole interface is influenced). Please see an example on how to apply this form of @text below in the examples."

Reply:
Fixed